.cirrus/tasks.yml

```diff
@@ -62,9 +63,9 @@ multi_arch_build_gcp_staging_task:
   only_if: $CIRRUS_CRON == 'nightly-mend-scan' || $TRIGGER == 'PUSH_GCP_STAGING_IMAGES'
   env:
     matrix:
-      - version: 10/datacenter/app
+      - version: 2025.1/datacenter/app
         STAGING_IMAGE_NAME: ${GCLOUD_STAGING_REGISTRY}/${GCLOUD_STAGING_PRODUCT_NAME}
-      - version: 10/datacenter/search
+      - version: 2025.1/datacenter/search
```

Comment:
should we put 2025.1 or 2025 only ? do you have a special opinion here ?

Reply:
That's a good point! I almost made it too generic but opted to use 2025.1 in a last-minute decision.
I thought that it would be easier to duplicate the code for 2025.x, than go from generic to multiple specific versions.